tests/func/test_dataset_query.py

```diff
@@ -1185,8 +1185,8 @@ def name_len_interrupt(_name):
 
     q = (
         DatasetQuery(name="animals", version=1, catalog=catalog)
-        .filter(C.size < 13)
-        .filter(C.parent.glob("cats*") | (C.size < 4))
+        # .filter(C.size < 13)
```

Comment:
disregard this, unrelated, this PR is primarily for @mattseddon and can be closed

Reply:
closed my PR, you can take it

Reply:
sounds good